### ISSUE TITLE
Support alterations to hold or remove columns of a result-set by name or index

### ISSUE THREAD

**Issue by Seddryck** [Seddryck/NBi#385]
It should be possible to define a way to remove or hold some columns in a result-set. The identification of the columns should be the index or the name. The order of the columns in the result is determined by their original order in the table. Only the columns that were specified as arguments are dropped. The other columns are included in the result. 

```xml
<alteration>
  <project-away>
    <column identifier="#1">
    <column identifier="Foo">
  </project-away>
</alteration>
```

the command to hold columns should be ```project```. The order of the columns in the result is specified by the order of the arguments. Only the columns specified in the arguments are included in the result: any others in the input are dropped.

**Comment by Seddryck** [comment]
Alpha version is available at https://ci.appveyor.com/project/Seddryck/nbi/build/1.18.0-alpha.1+161.build.428

**Comment by Seddryck** [comment]
Beta available in 1.21.0-beta0313
